client/src/components/authentication/login/login.tsx

```diff
@@ -26,22 +27,26 @@ const Login: React.FC = (): ReactElement => {
   }
 
   return (
-    <div className="login-container">
-      <div className="input-boxes">
-        <label>
-          <span>username</span>
-          <input {...username} />
-        </label>
-        <label>
-          <span>password</span>
-          <input {...password} />
-        </label>
-      </div>
-
-      <div className="submit-container">
-        <button type="button" data-test="submit-btn" onClick={handleSubmit}>
-          Submit
-        </button>
+    <div className="auth-container">
```

Comment:
I saw the layout for it. What do you think about adding a `Form` component?

Reply:
Talked offline. Will add in another branch